### DISCUSSION TITLE
Abnormal behaviour in connections

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx-operator#627] [General]
Seeing some abnormal behaviour in connections in emqx v5.0.11, cluster is having around 3000 connections, tried to kick out clients but all are reconnection immediately. Tried sending diconnect packets, response - Couldnt disconnect connection. Also tried restarting the cluster still reconnection is happening. Another strange thing is that many of the clients are connected with the same client id
What is the reason for this?
Anyway to disconnect all connections?

Thanks in advance......

**Comment by hjianbo** [comment]
Maybe you can try to add these connections to the ban list via HTTP API https://www.emqx.io/docs/en/v5.0/admin/api-docs.html#tag/Banned
